.zenodo.json

```diff
@@ -0,0 +1,21 @@
+{
+    "creators": [
+        {
+            "affiliation": "Netherlands eScience Center",
+            "name": "Renaud, Nicolas",
```

Comment:
I guess more people need to be added here....

Reply:
Indeed this was just an example I was doing with Nico